core/src/main/scala/org/genivi/sota/core/data/Campaign.scala

```diff
@@ -10,17 +10,19 @@ import eu.timepit.refined.string._
 import io.circe._
 import java.time.Instant
 import java.util.UUID
+
 import org.genivi.sota.core.SotaCoreErrors
 import org.genivi.sota.data.{Namespace, PackageId, Uuid}
 import slick.driver.MySQLDriver.api._
-
 import Campaign._
+import com.advancedtelematic.libats.codecs.{CirceEnum, SlickEnum}
+import org.genivi.sota.core.data.CampaignStatus.Status
 
 case class Campaign (meta: CampaignMeta, packageId: Option[PackageId], groups: Seq[CampaignGroup]) {
   def canLaunch(): Throwable Xor Unit = {
-    if (!meta.packageUuid.isDefined || groups.size < 1) {
+    if (meta.packageUuid.isEmpty || groups.size < 1) {
       Xor.Left(SotaCoreErrors.CantLaunchCampaign)
-    } else if (meta.launched) {
+    } else if (meta.status == CampaignStatus.Active) {
```

Comment:
Hmm, is this what we want? We can launch if we are in the following states:
- `Draft`
- `InPreparation`
- `Failed`
- `Finished`

Maybe it is better if the code is `meta.status != Campaign.Draft`?